src/DiagramGenerator/Diagram.php

```diff
@@ -150,7 +150,8 @@ public function draw()
             $this->image = $this->image->appendImages(true);
         }
 
-        $this->image->setImageFormat('png');
+        $this->image->setImageFormat($this->config->getImageFormat());
+        $this->image->setImageCompressionQuality($this->config->getImageCompressionQuality());
```

Comment:
I set the default value to 0 for png which is what i got by calling `getImageCompressionQuality`, i never tested with other values. I found this [stackoverflow post](http://stackoverflow.com/a/26997997/1924250)